### ISSUE TITLE
matplotlib: Cannot access member for type "Axes"

### ISSUE THREAD

**Issue by tekumara** [microsoft/pylance-release#147]
## Environment data

-   Language Server version: v2020.7.2
-   OS and version: macOS
-   Python version (& distribution if applicable, e.g. Anaconda): Python 3.6.10

## Expected behaviour

No type error. It should be possible to call `set_title`, `legend`, `set_xlabel`, `set_ylabel`, `set_ylim` on an subplot.
See https://matplotlib.org/3.2.2/api/_as_gen/matplotlib.pyplot.subplots.html

## Actual behaviour

```
Cannot access member "set_title" for type "Axes"
  Member "set_title" is unknown Pylance (reportGeneralTypeIssues)
```

## Code Snippet / Additional information

```python
import matplotlib.pyplot as plt

def plot(title: str = None):
    _, ax = plt.subplots(figsize=(10, 6))
    ax.set_title(title)
    ax.legend()
    ax.set_xlabel('X')
    ax.set_ylabel('Y')
    ax.set_ylim(0, 1.05)

    return ax
```

**Comment by tekumara** [comment]
`ax.collections` is another member I'm accessing.

**Comment by erictraut** [comment]
This is due to missing methods in the bundled matplotlib type stub. In "axes.pyi", note the comment:

```
    # TODO: Most methods on Axes are the same as pyplot
    # (as pyplot is just global functions onto a shared Axes).
    # Mirror those here.
```

**Comment by berkdemir** [comment]
Example:
![image](https://user-images.githubusercontent.com/40423004/108316683-492fc080-71ce-11eb-8f5b-f2d9e42fcfcf.png)

**Comment by jakebailey** [comment]
Largely, the original issue of Axes missing members is fixed. Most items have types, but some are still marked as `Any` and stand to be improved in the future. The next release has a couple of improvements, but at this point it's bug-level type changes, so I'm going to mark this as "fixed" in the next release.

We can still open other issues for specific changes that should be made.

RE: Figure, that error should no longer appear (as I've marked those stubs as incomplete, which I failed to do when I wrote them the first time over a year ago), and we can further refine it as well. I'm hoping to get these out to the public python-type-stubs repo to accept more changes as well (that just takes a little time).
